src/Microsoft.OData.Core/Uri/SelectExpandClauseToStringBuilder.cs

```diff
@@ -284,5 +298,29 @@ public override string Translate(ExpandedReferenceSelectItem item)
 
             return string.Concat(currentExpandClause, string.IsNullOrEmpty(res) ? null : string.Concat(ExpressionConstants.SymbolOpenParen, res, ExpressionConstants.SymbolClosedParen));
         }
+
+        /// <summary>
+        /// Translate an ExpandedCountSelectItem
+        /// </summary>
+        /// <param name="item">the item to Translate</param>
+        /// <returns>Defined by the implementer</returns>
+        public override string Translate(ExpandedCountSelectItem item)
+        {
+            NodeToStringBuilder nodeToStringBuilder = new NodeToStringBuilder();
+            string currentExpandClause = String.Join("/", item.PathToNavigationProperty.WalkWith(PathSegmentToStringTranslator.Instance).ToArray());
+            string res = string.Empty;
+            if (item.FilterOption != null)
+            {
+                res += "$filter=" + nodeToStringBuilder.TranslateFilterClause(item.FilterOption);

```

Comment:
@KenitoInc I think we have constants for `$filter` and `=` so we should probably use them here?